### PR TITLE
Documentation for auto skills

### LINKED ISSUE #200: Add Documentation for auto skills
**Actual Behaviour**
Currently there is no documentation about the current work on autoskills

Please state here what is currently happening.

**Expected Behaviour**
To add the required documentation

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by prateekiiest**
Fixes #200 

Changes

NA
Screenshots for the changes
NA

### CODE REVIEW

File: media_daemon/DAEMON.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The Media Discovery Daemon is the daemon that we are using to detect whether a U
 If a new USB connection is detected, the python script `auto_skills.py` is triggered which creates a custom skill in the SUSI server and allows the user to play music from the USB device. But if the USB device is removed, the skill file is removed and the server functions normally.
 <p/> 
 
+<p>
+The `autostart.sh` acts as a starting point for executing the `auto_skills.py` which in turn creates the custom skill under '/home/pi/SUSI.AI/susi_linux/media_daemon/' as custom_skill.txt . The custom skill contains a JSON response of the audio play skill with file identifiers USB name and mp3 file name.
```
Comment (stealthanthrax) [inline media_daemon/DAEMON.md]: @prateekiiest it's not `/home/pi`, we are using `$HOME` instead.

Comment (stealthanthrax) [reply media_daemon/DAEMON.md]: Also, the skill is created under `$HOME/SUSI.AI/susi_linux/susi_server/susi_server/data/generic_skills/media_discovery/`

**Comment by stealthanthrax** [comment]
@prateekiiest ,please see the inline comments

**Comment by prateekiiest** [comment]
made the changes @stealthanthrax

**Pull request merged**